setup.py

```diff
@@ -457,5 +457,6 @@ def run(self):
     zip_safe=False,
     scripts=["scripts/iyt"],
     ext_modules=cython_extensions + extensions,
-    python_requires='>=2.7,!=3.0.*,!=3.1.*,!=3.2.*,!=3.3.*,!=3.4.*'
+    python_requires='>=2.7,!=3.0.*,!=3.1.*,!=3.2.*,!=3.3.*,!=3.4.*',
+    package_data={'yt' : ['pkg_data/*.yaml']}
```

Comment:
Oh!  I think we still want the `yaml` files to be in the frontends directory, so this would be better as `yt/frontends/*/*.yaml` I think.